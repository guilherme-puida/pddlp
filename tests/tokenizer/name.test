name
name-hyphens
name_underscores
namedigit1
name-hypens-digit-10
name_underscores_digit12
name_with-all-3_elements
