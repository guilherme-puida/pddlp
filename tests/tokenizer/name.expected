[01:01] name
[02:01] name-hyphens
[03:01] name_underscores
[04:01] namedigit1
[05:01] name-hypens-digit-10
[06:01] name_underscores_digit12
[07:01] name_with-all-3_elements
EOF